### PR TITLE
CDPS-627: Create probation documents page

### PR DESCRIPTION
**Pull request by danielburnley**

### CODE REVIEW

File: integration_tests/pages/probationDocuments.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    const details = () => tableRow().find('.govuk-table__cell').eq(0)
+    return {
+      details: () => ({
+        documentLink: () => details().find('.govuk-grid-column-full').eq(0).find('a'),
+        documentTypeAndAuthor: () => details().find('.govuk-grid-column-full').eq(1),
+        documentDescription: () => details().find('.govuk-grid-column-full').eq(2),
+      }),
+      date: () => tableRow().find('.govuk-table__cell').eq(1),
+    }
+  }
```
Comment (brightonsbox) [inline integration_tests/pages/probationDocuments.ts:19]: These structures are really nice!

File: server/routes/index.ts [hunk 2]
```diff
@@ -227,6 +228,7 @@ export default function routes(services: Services): Router {
   router.use(professionalContactsRouter(services))
   router.use(goalsRouter(services))
   router.use(locationDetailsRouter(services))
+  router.use(probationDocumentsRouter(services))
```
Comment (brightonsbox) [inline server/routes/index.ts:231]: I think David mentioned that routes with only one related endpoint as we have here should be inline in this file, but I do like the idea of organising in separate files, even in this case - maybe worth another quick chat in our catch up on Monday?

**Review by brightonsbox — APPROVED** [review]
LGTM!

File: integration_tests/pages/probationDocuments.ts (hunk 1 above)
Comment (danielburnley) [reply integration_tests/pages/probationDocuments.ts:19]: creating nice page objects is always a satisfying thing to do :')

**Pull request merged**